plugins/get_url/unit_test/test_get_file.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+
+from unit_test.util import Util
+
+sys.path.append(os.path.abspath("../"))
+
+from unittest import TestCase
+from komand_get_url.actions.get_file import GetFile
+from komand_get_url.actions.get_file.schema import Input
+from unittest.mock import patch
+
+sys.path.append(os.path.abspath("../"))
+
+
+class TestGetFile(TestCase):
+    def tearDown(self):
+        Util.clear_cache()
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.action = Util.default_connector(GetFile())
+
+    @patch("six.moves.urllib.request.urlopen", side_effect=Util.mocked_request)
```

Comment:
This might be what is causing issues in your unit test runs check

Reply:
Could you tell more why this might causing issue? This line only mock request.

The error in unit test is:
```
raise KeyError(key) from None\nKeyError: \'SSL_CERT_FILE\'\n'
```

It looks like on OS there is no cert file. We are looking at this :)